test/parallel/test-listen-fd-detached-inherit.js

```diff
@@ -65,7 +65,6 @@ function test() {
 function parent() {
   var server = net.createServer(function(conn) {
     throw new Error('Should not see connections on parent');
-    conn.end('HTTP/1.1 403 Forbidden\r\n\r\nI got problems.\r\n');
```

Comment:
I guess that if we don't end the connection, the test would fail for timeout reasons.
It is better to see this error, I swapped the lines.